src/main/java/thePackmaster/cards/grandopeningpack/Lights.java

```diff
@@ -0,0 +1,84 @@
+package thePackmaster.cards.grandopeningpack;
+
+import com.evacipated.cardcrawl.mod.stslib.cards.interfaces.StartupCard;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInDrawPileAction;
+import com.megacrit.cardcrawl.actions.utility.ScryAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.EnergizedPower;
+import com.megacrit.cardcrawl.powers.StrengthPower;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.cards.evenoddpack.AbstractEvenOddCard.makeCardTextGray;
+import static thePackmaster.util.Wiz.atb;
+
+public class Lights extends AbstractGrandOpeningCard {
+    public final static String ID = makeID("Lights");
+
+    public Lights() {
+        super(ID, 1, CardType.SKILL, CardRarity.RARE, CardTarget.SELF);
+        baseMagicNumber = magicNumber = 2;
+        cardsToPreview = new Camera();
```

Comment:
You might want to use MultiCardPreview to preview both Camera and Action -- if you see Lights in a card reward right now, there's no way to know what Action does. There should be examples of using it elsewhere in Packmaster.

This is the same flaw that Alpha has in the base game, but thanks to MultiCardPreview we can do better!